vars/execIQETestsWithNotifier.groovy

```diff
@@ -32,15 +32,23 @@ def getCurrentMinusBuilds(number) {
     return currentList
 }
 
-def checkCurrentBuilds(buildList) {
+def checkResolved(buildList) {
+    int failures = 0
+
     for(build in buildList) { 
         if (build.getResult().toString() != "SUCCESS") {
             echo "build ${build.getDisplayName()}'s result was a ${build.getResult().toString()}"
-            return false
+            failures++;
         }
     }
 
-    return true
+    if buildList.first().getResult().toString() != "SUCCESS" && failures == 1 {
```

Comment:
Hi @skarekrow, what will happen if buildList includes only one test, and that test will have a failure?
Does your change cover such a situation?